File: app/src/main/java/com/wineguesser/deductive/model/wine/BarrelAge.java
package com.wineguesser.deductive.model.wine;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
class BarrelAge {
    private int ageOld;
    private int ageNew;

    public int getOld() {
        return ageOld;
    }

    public void setOld(int ageOld) {
        this.ageOld = ageOld;
    }

    public int getNew() {
        return ageNew;
    }

    public void setNew(int ageNew) {
        this.ageNew = ageNew;
    }
}

File: app/src/main/java/com/wineguesser/deductive/model/wine/BarrelAge.kt
package com.wineguesser.deductive.model.wine

data class BarrelAge(
    var ageOld: Int = 0,
    var ageNew: Int = 0
) {
    var old: Int
        get() = ageOld
        set(value) { ageOld = value }
    var new: Int
        get() = ageNew
        set(value) { ageNew = value }
}
